src/api/test-controller/index.js

```diff
@@ -209,6 +212,51 @@ export default class TestController {
         return this._enqueueCommand('dragToElement', DragToElementCommand, { selector, destinationSelector, options });
     }
 
+    _parseSelectorForScroll (args) {
+        const selector = typeof args[0] === 'string' || isSelector(args[0]) ? args[0] : null;
+
+        if (selector)
+            args.shift();
+        else
+            // eslint-disable-next-line no-undef
+            return () => document.scrollingElement || document.documentElement;
```

Comment:
For which cases do we need the `document.scrollingElement` expression?

Reply:
Safari does not work correctly with document.documentElement for some reason. I checked it on BrowserStack MacOS machines and real iPhone device

Reply:
Add comment please for that.